aix/AixProcess.c

```diff
@@ -0,0 +1,243 @@
+/*
+htop - AixProcess.c
+(C) 2015 Hisham H. Muhammad
+(C) 2018 Calvin Buckley
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "Process.h"
+#include "AixProcess.h"
+#include "CRT.h"
+#include <stdlib.h>
+
+/*{
+#include "Settings.h"
+#include <sys/corralid.h> // cid_t 
+
+#define Process_delete AixProcess_delete
+
+typedef enum AixProcessFields {
+   // Add platform-specific fields here, with ids >= 100
+   WPAR_ID = 100,
+   LAST_PROCESSFIELD = 101,
+} AixProcessField;
+
+typedef struct AixProcess_ {
+   Process super;
+   int kernel;
+   cid_t cid; // WPAR ID
+   unsigned long long int stime, utime; // System/User time (stored sep for calculations)
+} AixProcess;
+
+#ifndef Process_isKernelThread
+#define Process_isKernelThread(_process) (_process->kernel == 1)
+#endif
+
+#ifndef Process_isUserlandThread
+// XXX
+#define Process_isUserlandThread(_process) (false)
+#endif
+
+}*/
```

Comment:
This looks like it was from the header file and should be removed from the code.